code/style.css

```diff
@@ -1,7 +1,339 @@
+  font-family: 'Montserrat', sans-serif; 
+  width: 350px; 
+  height: auto;
+  padding: 30px 0px 30px 0px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.sign-up-form h3 {
+  text-align: center;
+}
+
+.checkboxes {
+  display: flex; 
+  flex-direction: column;
+}
+
+
+::placeholder {
+  font-size: 15px;
+  font-style: italic;
+}
+
+
+form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+
+input {
+  font-size: 20px;
+  margin: 15px;
+}
+
+
+label {
+  font-size: 15px; 
+}
+
+.text-labels {
+  display: block;
+  margin-left: 14px;
+}
+
+
+button {
+  border: 1px solid gray;
+  background-color:antiquewhite;
+  font-size: 17px;
+  width: 130px;
+  border-radius: 20px;
+  padding: 10px 15px 10px 15px;
+  margin-top: 10px;
+}
+
+button:hover {
+  background-color: #DECDB4;
+}
+
+/* -------------ACCORDION------------------ */
+
+.accordion {
+  border: 2px solid #DECDB4;
+  border-radius: 10px;
+  background-color: #DECDB4;
+  width: 100%;
+  height: auto;
+  font-family: 'Montserrat', sans-serif;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-bottom: 30px;
+}
+
+.answer {
+  border: 1px solid #F3E9E0;
+  width: 85%;
+  margin: 10px;
+  padding: 0px 20px 0px 20px;
+  text-align: justify;
+  display: none;
+}
 
 .active + .answer {
-  /* some styling here */
+  display: block;
+}
+  
+.question {
+  border-radius: 5px;
+  background-color: #F3E9E0; 
+  width: 90%;
+  padding: 10px 0px 10px 10px;
+  font-size: 20px;
+  margin-bottom: 10px;
+  margin-top: 10px;
+}
+
+/* This image below is only displayed in deskptop version that is why it displays none in this mobile first design */
+
+.small-image {
+  display: none;
+}
+
+/* --------------footer--------------- */
+
+footer {
+  background-color: #DECDB4;
+    position: relative;
+    font-size: 30px;
+    width: 100%;
+    padding-bottom: 20px;
+    height: 50px;
+    color: white; 
+    text-align: center;
+    object-fit: cover;
+    overflow: hidden;
+}
+
+
+.fa:hover {
+  color: #F3E9E0;
+  font-size: 40px;
+  text-decoration: none;
+}
+
+
+
+
+/* ------------TABLET MEDIA QUERIES-------------- */
+
+
+@media screen and (min-width: 768px) and (max-width: 1024px) {
+
+   .caption {
+       font-size: 25px;
+   }
+
+   .flex-container {
+      max-width: auto;
+      display: flex;
+      flex-direction: column;
+      column-gap: 4em;
+      object-fit: cover;
+      
+  }
+
+  .sign-up-form {
+    width: 100%; 
+    height: 500px;
+  }
+
+  .accordion {
+    width: 100%;
+    height: auto;
+    padding-bottom: 30px;
+  }
+
+}
+
+/* -------------DESKTOP MEDIA QUERIES-------------- */
+
+
+@media screen and (min-width: 1025px) {
+
+  .flex-container {
+      display: flex; 
+      flex-direction: row;
+      justify-content: center;
+      align-items: center;
+      column-gap: 2em;
+      flex-wrap: wrap;
+      overflow: hidden;
+      object-fit: cover;
+  }
+
+  .sign-up-form {
```

Comment:
The code has placed itself a bit wrong in this section =) (just need to tab it in)